models/template/inst/analysis_qsub.job

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -l
+module load R/4.1.2
```

Comment:
this is very server specific, in the pecan.xml we have a section for qsub that allows you to specify the modules that need to be loaded. See https://pecanproject.github.io/pecan-documentation/master/xml-core-config.html#xml-host for an example.